resources/views/battcatora/info-modal.blade.php

```diff
@@ -20,7 +20,7 @@
                 <p>@lang('battcatora.info.body-s3-p1', ['url' => 'https://openrepair.org/about/'])</p>
                 <hr>
                 <h5>@lang('battcatora.info.body-s4-header')</h5>
-                <p>@lang('battcatora.info.body-s4-p1', ['url' => 'https://talk.restarters.net/t/5030'])</p>
+                <p>@lang('battcatora.info.body-s4-p1', ['url' => env('DISCOURSE_URL') . 't/t/5030'])</p>
```

Comment:
This doesn't seem correct - it seems to have an extra 't'.

I think it should be: ['url' => env('DISCOURSE_URL') . '/t/5030'])</

Reply:
Yes, I spotted that and it should already be fixed.